### ISSUE TITLE
Accessible name of the close button that is not relevant

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/meet#864]
## Bug Report

**Problematic behavior**
The accessible name of the modal close button has an accessible name that is not relevant.

**Expected behavior/code**
The relevant code is as follows:

```html
<button type="button" tabindex="0" data-react-aria-pressable="true" aria-label="shareDialog.closeDialog" class="d_flex jc_center ai_center trs_background_200ms,_outline_200ms,_border-color_200ms cursor_pointer bd_1px_solid_transparent [&amp;[data-disabled]]:cursor_default [&amp;[data-disabled]]:c_gray.300 gap_0.5rem bdr_4 --square-padding_0 bg-c_none! fw_medium! c_primary.900 [&amp;[data-hovered]]:bg-c_none! [&amp;[data-hovered]]:bd-c_colorPalette.active! [&amp;[data-pressed]]:bg-c_primary.300 [&amp;[data-pressed]]:bd-c_currentcolor bd-c_none! " data-rac="" id="react-aria888922586-:r48:" style="user-select: none;"><svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="currentColor" class="remixicon "><path d="M11.9997 10.5865L16.9495 5.63672L18.3637 7.05093L13.4139 12.0007L18.3637 16.9504L16.9495 18.3646L11.9997 13.4149L7.04996 18.3646L5.63574 16.9504L10.5855 12.0007L5.63574 7.05093L7.04996 5.63672L11.9997 10.5865Z"></path></svg></button>
```
**Steps to Reproduce**
1. Click on create a new meeting
2. This opens a new window
3. When using a screen reader, the focus is placed on the modal title "Your meeting is ready!"
4. Tab once more, and you land on the modal close button.

**Possible Solution**
Correct the aria-label="shareDialog.closeDialog" attribute with the label that should be corrected to "Close modal window" (in French and equivalent translations)

**Additional context/Screenshots**
Add any other context about the problem here. If applicable, add screenshots to help explain.

<img width="472" height="187" alt="Image" src="https://github.com/user-attachments/assets/8bfa5758-7e81-4a1d-99a6-15aa875fe452" />
